doc/voorstel/voorstel-inhoud.tex

```diff
@@ -23,9 +23,9 @@ \section{Literatuurstudie}%
 \label{sec:litaratuurstudie}
 
 Volgens een studie uit 2022~\autocite{Kotenko2022} is het opstellen van een configuratie inventaris essentieel voor het beoordelen van cybersecurityrisico's.
-Aan de hand van een inventaris kan men potenti\"ele kwetsbaarheden gaan identificeren en de passende maatregelen gaan treffen.
-Ook wordt er benadrukt dat het inventaris kan gebruikt worden om mogelijke cyberaanvalspaden te identificeren en het voorkomen van verliezen door cyberaanvallen.
-Het regelmatig herevalu\"eren van het inventaris wordt benadrukt als een proactieve benadering om potenti\"ele incidenten te voorkomen.
+Aan de hand van een inventaris kunnen potenti\"ele kwetsbaarheden ge\"identificeerd worden en passende maatregelen getroffen worden.
+Ook wordt er benadrukt dat de invenaris kan gebruikt worden om mogelijke cyberaanvalspaden te identificeren en het voorkomen van verliezen door cyberaanvallen.
+Het regelmatig herevalu\"eren van de invenaris wordt benadrukt als een proactieve benadering om potenti\"ele incidenten te voorkomen.
```

Comment:
I had a discussion on this with my better half (who is great in language since she used to write for newspapers) and it is most certainly 'de inventaris'. It also sounded strange to me but that's just the way it is. No need to merge it indeed. Maybe something to consider later on though.